lib/change_stream.js

```diff
@@ -74,6 +74,7 @@ class ChangeStream extends EventEmitter {
         ? new MongoDBNamespace('admin')
         : changeDomain.s.namespace;
 
+    this.parent = changeDomain;
```

Comment:
nit: since we're now creating the concept of a `parent` here, let's rename the `changeDomain` parameter to `parent`